pkg/apis/projectcalico/v3/globalalerttemplate.go

```diff
@@ -27,22 +27,7 @@ type GlobalAlertTemplate struct {
 	// Standard object's metadata.
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 	// Specification of the GlobalAlert.
-	Spec GlobalAlertTemplateSpec `json:"spec,omitempty"`
```

Comment:
I'm not sure I understand this part - what's the motivation for removing the GlobalAlertTemplateSpec?

Reply:
We went back to using a common `GlobalAlertSpec` for `GlobalAlert` and `GlobalAlertTemplate`. We don't need two different `Spec` objects when one is a template for the other.